coap_main.c

```diff
@@ -192,7 +192,7 @@ static CoAP_Result_t _rom SendResp(CoAP_Interaction_t* pIA, CoAP_InteractionStat
 
 		pIA->State = nextIAState; //move to next state
 
-		CoAP_EnqueueLastInteraction(pIA); //(re)enqueue interaction for further processing//todo: in die �u�ere statemachine
+		CoAP_EnqueueLastInteraction(pIA); //(re)enqueue interaction for further processing//todo: in die �u�ere statemachine
```

Comment:
No major issue and we should really replace German comments, but it seems there is some encoding issue. Can you just double check if all files are UTF-8 ?